PowershellTools.Common/Constants.cs

```diff
@@ -95,5 +95,10 @@ internal static class Constants
         /// PowerShell host process window title format
         /// </summary>
         public const string HostProcessWindowTitleFormat = "{0} : {1}";
+
+        /// <summary>
+        /// Error message for attempted to attach on PowerShell v4 or below

```

Comment:
No? Comment is trying to say that the error message is used when a user running v4 or below (since v5 introduced debug-runspace) tries to attach to a process.